tests/test_axis_limits.py

```diff
@@ -60,6 +60,10 @@
     "gbdrift",
     "cvdrift",
     "grad(alpha)",
+    "g^aa",
+    "g^ra",
+    "gbdrift",
+    "cvdrift",
```

Comment:
cvdrift is finite, even without the adding in $2\rho$

Reply:
Done.